src/hacbs/components/PipelineRunDetailsView/PipelineRunDetailsView.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import { NamespaceContext } from '../../../components/NamespacedPage/NamespacedPage';
+import { PipelineRunGroupVersionKind } from '../../models';
+import { PipelineRunKind } from '../../types';
+import DetailsPage from '../ApplicationDetails/DetailsPage';
+import PipelineRunDetailsTab from './tabs/PipelineRunDetailsTab';
+import PipelineRunEnterpriseContractTab from './tabs/PipelineRunEnterpriseContractTab';
+import PipelineRunEventsTab from './tabs/PipelineRunEventsTab';
+import PipelineRunLogsTab from './tabs/PipelineRunLogsTab';
+import PipelineRunTaskRunsTab from './tabs/PipelineRunTaskRunsTab';
+import PipelineRunYamlTab from './tabs/PipelineRunYamlTab';
+
+type PipelineRunDetailsViewProps = {
+  pipelineRunName: string;
+};
+
+export const PipelineRunDetailsView: React.FC<PipelineRunDetailsViewProps> = ({
+  pipelineRunName,
+}) => {
+  const { namespace } = React.useContext(NamespaceContext);
+  const navigate = useNavigate();
+
+  const [pipelineRun, loaded] = useK8sWatchResource<PipelineRunKind>({
+    groupVersionKind: PipelineRunGroupVersionKind,
+    name: pipelineRunName,
+    namespace,
+  });
+
+  if (!loaded) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
+
+  return (
+    <>
+      <React.Fragment>
+        <DetailsPage
+          breadcrumbs={[
+            { path: '/app-studio/pipelineruns', name: 'Pipelineruns' },
+            {
+              path: `/app-studio/applications/:${pipelineRunName}`,
+              name: pipelineRunName,
+            },
+          ]}
+          title={pipelineRunName}
+          actions={[
+            {
+              key: 'add-component',
+              label: 'Add Component',
+              onClick: () => {
+                navigate(`/app-studio/import?application=${pipelineRunName}`);
+              },
+            },
+            {
+              key: 'rerun',
+              label: 'Rerun',
+              href: '',
+              isDisabled: true,
+            },
+          ]}
+          tabs={[
+            {
+              key: 'detail',
+              label: 'Details',
+              component: <PipelineRunDetailsTab pipelineRun={pipelineRun} />,
```

Comment:
Not sure i understand... for details tab i'm passing the pipelineruns object. For the other tabs, i didn't change application name because they are just placeholders but i will change those to pass the object as per your other comment.

Reply:
I must have put my comment on the wrong line. Here's what the code looked like before.

`pplicationName={pipelineRunName}`

![image](https://user-images.githubusercontent.com/14068621/179760906-17379e65-f950-4d96-8eda-0b0352adcce4.png)